cql3/statements/alter_keyspace_statement.cc

```diff
@@ -101,6 +101,8 @@ cql3::statements::alter_keyspace_statement::prepare_schema_mutations(query_proce
         auto ks_md_update = _attrs->as_ks_metadata_update(ks_md, tm, feat);
         std::vector<mutation> muts;
         std::vector<sstring> warnings;
+        auto rf_map = _attrs->get_replication_map();
```

Comment:
nit: should have been defined in the previous patch

Reply:
Done, thanks.